src/scss/mixins/_date.scss

```diff
@@ -0,0 +1,33 @@
+@mixin oFormsDateFeatures($class) {
+    .#{$class}--date .#{$class}__group {
+        // We want to use `box-sizing: border-box` to set a max-height which includes padding.
+        // However when setting width to an `ex` value we want padding to be in addition.
+        $padding-width: $_o-forms-field-default-padding-leftright * 2;
+        margin-top: oTypographySpacingSize(2);
+
+        .#{$class}__label {
+            @include oFormsGroupLabel();
+            margin-bottom: oTypographySpacingSize(1);
+        }
+
+        .#{$class}__text {
+            margin-top: 0;
+        }
+
+        .#{$class}__text:first-of-type {
+            clear: left;
+        }
+
+        .#{$class}__text,
+        .#{$class}__label {
+            width: calc(2.5ex + #{$padding-width});
+            margin-right: $_o-forms-spacing;
+            float: left;
+        }
+
+        .#{$class}__text--year,
+        .#{$class}__label--year {
+            width: calc(5ex + #{$padding-width});
```

Comment:
Do we use `ex` anywhere else? It seems a bit unusual to introduce it to this component, and to the dates in this component alone 🤔

Reply:
As they're inline the width for these inputs is a little different. The width indicates what kind of data we're expecting, 2 chars for dd/mm or 4 chars for yyyy. But come to think of it should I be using `ch` rather than `ex` here. 🤔 GDS do the latter.

Reply:
Okay I've used to use `ch` (despite the new commit message saying the opposite). https://github.com/Financial-Times/o-forms/pull/231/commits/d04422e558741b634037fd0e3d9d06bbb33c0ea6#diff-53809cd931a4317a557310479fbe30ceR23